src/makeInfinityAdder.js

```diff
@@ -5,6 +5,23 @@
  */
 function makeInfinityAdder() {
   // write code here
+  let sum = 0;
+
+  function adder(arg) {
+    if (arg === undefined) {
+      const result = sum;
+
+      sum = 0;
+
+      return result;
+    } else {
+      sum += arg;
```

Comment:
It's good that you're accumulating the sum correctly. Just make sure to handle cases where something other than a number or `undefined` is passed to avoid any runtime errors.